exporter/lokiexporter/README.md

```diff
@@ -47,13 +51,16 @@ loki:
   endpoint: http://loki:3100/loki/api/v1/push
   tenant_id: "example"
   labels:
-    attributes:
+    resource:
       # Allowing 'container.name' attribute and transform it to 'container_name', which is a valid Loki label name.
       container.name: "container_name"
       # Allowing 'k8s.cluster.name' attribute and transform it to 'k8s_cluster_name', which is a valid Loki label name.
       k8s.cluster.name: "k8s_cluster_name"
+    attributes:
       # Allowing 'severity' attribute and not providing a mapping, since the attribute name is a valid Loki label name.
       severity: ""
+      rquest: "urlofrequest" 
```

Comment:
Apart from the misspelling of request, perhaps use a more typical example?  An access log containing http requests would use semantic conventions such as http.url, but this would not be suitable for a loki label.  http.host or http.status_code might make suitable loki labels though.
```suggestion
      http.status_code: "http_status_code" 
```